packages/at_secondary_server/lib/src/enroll/enroll_datastore_value.dart

```diff
@@ -15,6 +15,7 @@ class EnrollDataStoreValue {
   late String apkamPublicKey;
   EnrollRequestType? requestType;
   EnrollApproval? approval;
+  DateTime? expiresAt;
```

Comment:
enrollment is stored as AtData in our keystore. can we make use of AtMetaData.expiresAt

Reply:
fixed as part of [fbdafe4](https://github.com/atsign-foundation/at_server/pull/1536/commits/fbdafe4e55ca0a130f7e004f143d7f74d7437942). reverted the introduction of expiredAt in EnrollDataStoreValue